lib/Sema/TypeCheckMacros.cpp

```diff
@@ -49,51 +49,44 @@ using namespace swift;
 extern "C" void *swift_ASTGen_resolveMacroType(const void *macroType);
 extern "C" void swift_ASTGen_destroyMacro(void *macro);
 
-extern "C" void swift_ASTGen_freeString(const char *str);
+extern "C" void swift_ASTGen_freeBridgedString(BridgedString);
 
-extern "C" void *swift_ASTGen_resolveExecutableMacro(
-    const char *moduleName, ptrdiff_t moduleNameLength,
-    const char *typeName, ptrdiff_t typeNameLength,
-    void * opaquePluginHandle);
+extern "C" void *swift_ASTGen_resolveExecutableMacro(const char *moduleName,
+                                                     const char *typeName,
+                                                     void *opaquePluginHandle);
 extern "C" void swift_ASTGen_destroyExecutableMacro(void *macro);
 
 extern "C" ptrdiff_t swift_ASTGen_checkMacroDefinition(
-    void *diagEngine,
-    void *sourceFile,
-    const void *macroSourceLocation,
-    char **expansionSourcePtr,
-    ptrdiff_t *expansionSourceLength,
-    ptrdiff_t **replacementsPtr,
-    ptrdiff_t *numReplacements
-);
+    void *diagEngine, void *sourceFile, const void *macroSourceLocation,
```

Comment:
Suppose this PR is cleaning up string bridging, but another PR could be "cleanup bridging" for these cases as well (ie. to use `Bridged*`).